hr_expense_cancel/hooks.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+
+        for sheet in env['hr.expense.sheet'].search([]):
+            if sheet.expense_line_ids[0].payment_mode == 'own_account':
+                aml = sheet.account_move_id.mapped('line_ids')
+                reconcile = aml.mapped('full_reconcile_id')
+                aml_payment = reconcile.reconciled_line_ids.filtered(
+                    lambda r: r.id not in aml.ids)
```

Comment:
This can be simplified as `lambda r: r not in aml)` (and the variables is better called `amls`)

Reply:
done